examples/v2/chat2.nim

```diff
@@ -68,9 +83,23 @@ proc connectToNodes(c: Chat, nodes: seq[string]) {.async.} =
   c.connected = true
 
 proc publish(c: Chat, line: string) =
-  let payload = cast[seq[byte]](line)
-  let message = WakuMessage(payload: payload, contentTopic: DefaultContentTopic)
-  c.node.publish(DefaultTopic, message)
+  when PayloadV1:
+    # Use Waku v1 payload encoding/encryption
+    let
+      payload = Payload(payload: line.toBytes(), symKey: some(DefaultSymKey))
+      version = "1"
```

Comment:
Version was changed to integer in other PR I believe

Reply:
Yup, aware of this :)